docs/meetings/community/2022-11-15.md

```diff
@@ -0,0 +1,113 @@
+---
+tags: binder, meeting, notes
+---
+
+# JupyterHub and BinderHub Team Meeting
+
+- **Date:** Tuesday 15th November 2022
+- **Time:** 6PM UTC
+  - **Your timezone:** https://arewemeetingyet.com/UTC/2022-11-15/18:00/Hubs-Team-Meeting
+- **Video conference link:** https://meet.jit.si/HubsTeamMeeting
+- **This HackMD:** https://hackmd.io/@sgibson91/hubs-team-meeting
+- **GitHub issue:** https://github.com/jupyterhub/team-compass/issues/576
+- **Calendar for future meetings:** https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings/index.html#meeting-calendar
+
+#### By participating in this meeting, you are agreeing to abide by and uphold the [Code of Conduct](https://jupyter.org/conduct). Please take a second to read through it! :pray: 
+
+## Welcome to the Team Meeting
+
+Hello!
+
+If you are joining the team video meeting, sign in below so we know who was here. Roll call:
+
+- name / institution / GitHub handle
+- Christian Dike/ - /@christiandike
+- Saka Abass / Outreachy / Olaedey
+- Min RK / Simula / @minrk
+- Arafat Abdussalam / Outreachy / ArafatAbdussalam
+- Simon Li / University of Dundee / @manics
+- Sarah Gibson / 2i2c / sgibson91
+- Sylvain Corlay / QuantStack / SylvainCorlay / sylvain.corlay@quantstack.net
+- A T Darian / QuantStack / @afshin / afshin.darian@quantstack.net
+- Gabriela Vives / Quantstack / GabrielaVives / gabriela.vives@quantstack.net
```

Comment:
I can't seem to add a suggestion 😢 But let's remove the emails from the official record? I'll make sure to pass them onto the 2i2c folk

Reply:
+1 at least so that we don't we don't cause bot-spam for people

Reply:
I've forced pushed them away. Do we need to make it clearer in the template that people shouldn't add their emails? Or should people be free to add them if they want to be contacted?

Reply:
I think this is a one-off. I explicitly asked them to add some contact details so 2i2c could reach out regarding collaborating on UI/UX research regarding Binder launching, and adding them to the hackmd was the quickest way to transfer that info in the moment.